### ISSUE TITLE
Contrast on highlighted pokemon

### ISSUE THREAD

**Issue by tywy87** [Pokemon-x-Blackjack/pokemon-blackjack#67]
Contrast failed accessibility. If you want to pass with red as your background, try: #F4B4AE 

![Screen Shot 2023-04-11 at 11 15 27 AM](https://user-images.githubusercontent.com/109436128/231216715-e59886be-dd4b-401e-ad84-54e726460877.png)
![Screen Shot 2023-04-11 at 11 24 38 AM](https://user-images.githubusercontent.com/109436128/231216731-29f4c0f5-27e1-4094-94bc-d4277754e2d0.png)
